contracts/sol6/Dao/mock/KyberFeeHandlerWrapper.sol

```diff
@@ -0,0 +1,228 @@
+pragma solidity 0.6.6;
+
+import "../../utils/Utils5.sol";
+import "../../utils/zeppelin/ReentrancyGuard.sol";
+import "../../utils/zeppelin/SafeERC20.sol";
+import "../../utils/zeppelin/SafeMath.sol";
+import "../../IKyberDao.sol";
+import "../../IKyberFeeHandler.sol";
+import "../DaoOperator.sol";
+
+interface ExtendedIKyberFeeHandler is IKyberFeeHandler {
```

Comment:
Seems wrong format for interface name, could be `IKyberFeeHandlerExtension` or `IExtendedKyberFeeHandler`

Reply:
Similar for contract name, we are using `Wrap` as prefix for wrapper contracts

Reply:
I think all interfaces should start with I

till now we make sure to do that.

Reply:
Ok, going with `IFeeHandler`